### PR TITLE
migrate push_dense_parameters method

### PR DESCRIPTION
**Pull request by QiJune**

### CODE REVIEW

**Comment by skydoorkai** [comment]
Unittest fails.

**Comment by QiJune** [comment]
@skydoorkai Fixed.

File: elasticdl/python/worker/ps_client.py [hunk 1]
```diff
@@ -60,3 +63,32 @@ def pull_embedding_vectors(self, layer_name, embedding_ids):
         new_embeddings = np.empty_like(embeddings)
         new_embeddings[index] = embeddings
         return new_embeddings
+
+    def patition_dense_parameters(self, param_names):
```
Comment (skydoorkai) [inline elasticdl/python/worker/ps_client.py]: `patition_dense_parameters` -> `partition_dense_parameters`

Comment (QiJune) [reply elasticdl/python/worker/ps_client.py]: Done.

File: elasticdl/python/worker/ps_client.py [hunk 2]
```diff
@@ -14,14 +14,19 @@
...
+)
 
 
 class PSClient(object):
     def __init__(self, ps_stubs):
-        self._ps_stubs = ps_stubs
-        self._ps_num = len(self._ps_stubs)
+        self.ps_stubs = ps_stubs
+        self.ps_num = len(self.ps_stubs)
+        self.parameter_to_ps = {}
```
Comment (brightcoder01) [inline elasticdl/python/worker/ps_client.py:28]: dense_parameter_to_ps?

**Review by skydoorkai — APPROVED** [review]

**Pull request merged**